app/lib/blog/feed.rb

```diff
@@ -0,0 +1,30 @@
+module Blog
+  class Feed
+    include ActiveModel::Model
+
+    attr_accessor :title, :link, :latest_post_at, :posts
+
+    HEADWAY_URL = "https://headwayapp.co/rdv-service-public-changelog".freeze
+    CACHE_KEY = "blog_feed".freeze
+
+    def self.new_content_for_agent?(agent)
+      feed.latest_post_at < agent.blog_read_at
```

Comment:
je pense que cette condition est à l’envers, non ?

```suggestion
      feed.latest_post_at > agent.blog_read_at
```

edit : en fait je crois aussi que `feed` n’existe pas ici je suis un peu perplexe

edit 2 : et oui, après vérif quand on retire le rescue StandardError on voit que ça ne marche pas 🙈 j’allais d’ailleurs suggérer dans la foulée de supprimer ce rescue

Reply:
> je pense que cette condition est à l’envers, non ?

Absolument ! :grimacing: 

> edit : en fait je crois aussi que `feed` n’existe pas ici je suis un peu perplexe

En remplaçant `feed` avec `from_cache` ça fonctionne. :wink: Ceci étant dit, j'étais tenté de mettre en cache `latest_post_at` dans une entré de cache dédiée. Mais avec le Singleton on aura peut-être pas besoin de tout ça !

> edit 2 : et oui, après vérif quand on retire le rescue StandardError on voit que ça ne marche pas 🙈 j’allais d’ailleurs suggérer dans la foulée de supprimer ce rescue

Je pense qu'il faut garder un rescue quelque-part, car sinon pète toutes les pages de la plateforme si on ne trouve pas les news (une feature non essentielle).